test/e2e/qat/qatplugin_dpdk.go

```diff
@@ -251,6 +255,39 @@ func runCpaSampleCode(ctx context.Context, f *framework.Framework, runTests int,
 	gomega.Expect(err).To(gomega.BeNil(), utils.GetPodLogs(ctx, f, pod.ObjectMeta.Name, pod.Spec.Containers[0].Name))
 }
 
+func runQatEngineTestApp(ctx context.Context, f *framework.Framework, resourceName v1.ResourceName) {
+	ginkgo.By("submitting a pod requesting QAT" + resourceName.String() + "resources")
```

Comment:
we have systematic problems with the log generation:
```suggestion
	ginkgo.By("submitting a pod requesting QAT " + resourceName.String() + " resources")
```